scipy/stats/_continuous_distns.py

```diff
@@ -3501,12 +3497,23 @@ def _pdf(self, x, mu):
     def _logpdf(self, x, mu):
         return -0.5*np.log(2*np.pi) - 1.5*np.log(x) - ((x-mu)/mu)**2/(2*x)
 
+    def _logcdf(self, x, mu):
+        fac = np.sqrt(1 / x)
+        a = _norm_logcdf(fac * (x - mu) / mu)
+        b = 2 / mu + _norm_logcdf(-fac * (x + mu) / mu)
+        return a + np.log1p(np.exp(b - a))
+
+    def _logsf(self, x, mu):
+        fac = np.sqrt(1 / x)
+        a = _norm_logsf(fac * (x - mu) / mu)
+        b = 2 / mu + _norm_logcdf(-fac * (x + mu) / mu)
```

Comment:
```suggestion
        b = 2 / mu + _norm_logcdf(-fac * (x + mu) / mu)
```
I imagine this should be `_norm_logsf`, if it is defined.

Reply:
No, the paper uses the logcdf. Only the variable `a` uses logsf.

Reply:
Interesting. OK.